docs/frontend/styleguide.md

```diff
@@ -0,0 +1,45 @@
+# Shopsys Framework styleguide
```

Comment:
```suggestion
# Shopsys Framework Styleguide
```
because: All words in a title except conjunctions (and, or, but...), articles (a, an, the), and short prepositions (in, to, of...) should be capitalized. Other headings should not be capitalized.

In index.md it is named as Basic Design Styleguide, that might be better naming

Reply:
From our documentation:
All documentation files should be named same as first heading (in lowercase, non-alphanumeric characters replaced by dash).